src/_includes/content/boiler.md

```diff
@@ -0,0 +1,24 @@
+## Introduction
+
+B01LER is a fully customisable design system (DS) that enables you to easily create cohesive and consistent digital products across multiple platforms and brands.
+
+B01LER provides pre-built components that exist in design in the form of a [component library in Figma](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=8941%3A1818&mode=design&t=owVepZzAwdrehnmL-1) (the link to the Figma file above is exclusively for members of the Accenture organization, while those outside the organization can download the file [here](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig)), as well as in [code in a repository](https://github.com/deven-org/B01LER-Kitchen), which is also [displayed in the web using Storybook](https://b01ler.onrender.com/). Both the components in design and in code are connected via a token system.
+
+A token system is a collection of values which represent the visual properties of a design system, such as typography, color and spacing.
+
+Utilizing tokens, B01LER is simultaneously consistent and flexible. By changing the tokens, you have the power to change the appearance of the whole design, including all the components and all the digital products build with it. In Figma and in code.
+
+Overall, B01LER is a powerful tool that helps you work smarter, not harder. By streamlining the design and development processes B01LER empowers you to create better products faster.
+
+### Benefits of B01LER
+1. Increased efficiency in building design systems on brand, because semantic structure and architecture are pre-built
+2. Increased efficiency, through automated processes. Documentation, DEV Handover, and more
+3. Increased effectivity, because the time saved on building all the components can be used to build features right away
+4. Increased acceptance of DS because components work right from the beginning of project phase
+5. Immense cost savings, because the DS ramp-up phase is reduced from multiple months to days
+
+## Resources
+- [Browse through our website](https://boilerds.com/)
+- [Explore our Figma File](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=8941%3A1818&mode=design&t=owVepZzAwdrehnmL-1) (the link to the Figma file above is exclusively for members of the Accenture organization, while those outside the organization can download the file [here](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig))
```

Comment:
The public link should probably be the main one:

```suggestion
- [Download our Figma File](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig) (Accenture-internal: [View on Figma](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=8941%3A1818&mode=design&t=owVepZzAwdrehnmL-1))
```

Reply:
Again a good point, will update the content